contrib/electrs/build_electrs.py

```diff
@@ -75,7 +75,7 @@ def verify_repo(allow_modified):
         logging.error("Validation failed - %s has local modifications.", ELECTRS_DIR)
         allow_modified or bail("Bailing")
 
-    if repo.head.object.hexsha != EXPECT_HEAD:
+    if EXPECT_HEAD != None and repo.head.object.hexsha != EXPECT_HEAD:
```

Comment:
I'd like to see a warning logged when this is skipped

Reply:
done